github/repos.go

```diff
@@ -573,40 +572,13 @@ type PullRequestReviewsEnforcement struct {
 // because the request structure is different from the response structure.
 type PullRequestReviewsEnforcementRequest struct {
 	// Specifies which users and teams should be allowed to dismiss pull request reviews. Can be nil to disable the restrictions.
-	DismissalRestrictionsRequest *DismissalRestrictionsRequest `json:"dismissal_restrictions"`
+	DismissalRestrictionsRequest *DismissalRestrictionsRequest `json:"dismissal_restrictions,omitempty"`
```

Comment:
You said:

> For personal repositories, the 'dismissal_restrictions' must be omitted.

I think it's worth documenting here as well. We can take the phrasings from GitHub API docs:

> User and team dismissal restrictions are only available for organization-owned repositories. Omit this parameter for personal repositories.